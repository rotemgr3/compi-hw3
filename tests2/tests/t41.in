 
//piazza 103 - should be ok +
override void foo(int x, bool y) {int booga = 5;}
override void foo(bool x, int y) {int booga = 5;}
void main() {int booga = 5;}
