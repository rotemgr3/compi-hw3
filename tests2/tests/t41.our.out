---end scope---
x INT -1
y BOOL -2
booga INT 0
---end scope---
x BOOL -1
y INT -2
booga INT 0
---end scope---
booga INT 0
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
foo (INT,BOOL)->VOID 0
foo (BOOL,INT)->VOID 0
main ()->VOID 0
